mybib.bib

```diff
@@ -1,194 +1,219 @@
+  note         = {This is a note.},
+  langid       = {czech},
 }
 
-@ARTICLE{big,
-  author = {Jass, Hugh},
-  title = {A big paper},
-  journal = {The journal of big papers},
-  year = {1991},
-  volume = {23}
+@article{big,
+  author       = {Jass, Hugh},
+  title        = {A big paper},
+  journaltitle = {The journal of big papers},
+  date         = {1991},
+  volume       = {23},
 }
 
-@ARTICLE{cs,
+@article{cs,
   COMMENT_OUTauthor = {Černý, Hugh},
-  title = {Titulek},
-  journal = {The journal of big papers},
-  year = {1991},
-  volume = {12},
-  pages = {22-44},
-  doi = {10.112.22/jkn},
-  number = {2},
-}
-
-@BOOK{koll,
-  author = {Kollmannová, Ludmila and Bubeníková, Libuše and Kopecká, Alena},
-  title = {Angličtina pro samouky}, 
-  edition = {5},
-  location = {Praha}, 
-  publisher = {Státní pedagogické nakladatelství}, 
-  year = {1977}, 
+  title        = {Titulek},
+  journaltitle = {The journal of big papers},
+  date         = {1991},
+  volume       = {12},
+  number       = {2},
+  pages        = {22--44},
+  doi          = {10.112.22/jkn},
+}
+
+@book{koll,
+  author    = {Kollmannová, Ludmila and Bubeníková, Libuše and Kopecká, Alena},
+  title     = {Angličtina pro samouky},
+  edition   = {5},
+  location  = {Praha},
+  publisher = {Státní pedagogické nakladatelství},
+  date      = {1977},
   pagetotal = {525},
-  series = {Učebnice pro samouky},
-  number = {4},
-  isbn = {80-04-25663-5}
+  series    = {Učebnice pro samouky},
+  number    = {4},
+  isbn      = {80-04-25663-5},
 }
 
-@INBOOK{kap,
-  author = {Novotná, Pepina},
-  title = {Podkapitola},
-  pages = {22-29},
-  chapter = {2.},
-  crossref = {koll}
+@inbook{kap,
+  author   = {Novotná, Pepina},
+  title    = {Podkapitola},
+  pages    = {22--29},
+  chapter  = {2.},
+  crossref = {koll},
 }
 
-@PERIODICAL{tug,
+@periodical{tug,
   journaltitle = {TUGBoat},
-  publisher = {TUG},
-  issn = {1222-3333},
-  langid = {english},
-  date = {1980/},
-  options = {skipbib=true},
-  url={http://tugboat.tug.org/}
-}
-
-@ARTICLE{knuth,
-  author = {Knuth, Donald},
-  title = {Journeys of \TeX},
-  volume = {17},
-  number = {3},
-  year = {2003},
-  pages = {12-22},
-  url = {http://tugboat.tug.org/kkk.pdf},
-  crossref={tug}
-}
-
-@COLLECTION{sbornik,
-title = {Mimořádně užitečný sborník},
-editor = {Geniální, Jiří},
-year = {2007},
-langid = {czech},
-location = {Praha},
-publisher = {Academia},
-isbn = {978-222-626-222-2}
-}
-@INCOLLECTION{prispevek1,
-crossref = {sbornik},
-title = {Velmi zajímavý článek},
-author = {Vlaštovka, Josef},
-pages = {22-45}
+  publisher    = {TUG},
+  date         = {1980/},
+  issn         = {1222-3333},
+  url          = {http://tugboat.tug.org/},
+  langid       = {english},
+  options      = {skipbib=true},
+}
+
+@article{knuth,
+  author   = {Knuth, Donald},
+  title    = {Journeys of \TeX},
+  date     = {2003},
+  volume   = {17},
+  number   = {3},
+  pages    = {12--22},
+  url      = {http://tugboat.tug.org/kkk.pdf},
```

Comment:
I might be overly sensitive, but I find the [abbreviation `kkk` problematic](https://en.wikipedia.org/wiki/Ku_Klux_Klan). This appears to be a made up article anyway and the link doesn't work. Maybe a real article with a working link might be more appropriate. (I won't duplicate the comment, but the entry is also used in the `.tex` file)

Reply:
I think there are more problematic parts, like invalid ISBN or DOI and URLs as well. The intuition was pure semantic I would say (like `pokus.cz` is `example.com`), but 3xk was probably not a good choice. I'll try to look at this later on as well.

Reply:
Fixed a bit in 6faef1582b627c537e131211a9db358c2843a9c6